app/serializers/search/postgres_podcast_episode_serializer.rb

```diff
@@ -11,16 +15,14 @@ class PostgresPodcastEpisodeSerializer < ApplicationSerializer
     attribute :hotness_score, -> { 0 }
 
     attribute :main_image do |podcast_episode|
-      Images::Profile.call(podcast_episode.podcast.profile_image_url, length: 90)
+      podcast_image_url(podcast_episode)
     end
 
     attribute :podcast do |podcast_episode|
-      podcast = podcast_episode.podcast
-
       {
-        slug: podcast.slug,
-        image_url: podcast.image_url,
-        title: podcast.title
+        slug: podcast_episode.podcast_slug,
+        image_url: podcast_image_url(podcast_episode),
+        title: podcast_episode.title
```

Comment:
I don't know of another way to avoid calling `Images::Profile.call` twice short of passing the result in as a param from the service...that felt pretty hacky 🤔

Reply:
I think you could push that decision into an instance method - maybe a memoizing method on the `podcast_episode` since that's the instance you are passing already? 

```
class PodcastEpisode
  def podcast_image_url
    @profile_image ||= Images::Profile.call(podcast.profile_image_url, length: 90)
  end
end
```

Agree there's not an immediately attractive way to do that inside the serializer (at the class level).